scripts/typescript/tsconfig.fluentui.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "module": "commonjs",
     "target": "es5",
-    "lib": ["es2015", "dom"],
+    "lib": ["es2019", "dom"],
```

Comment:
To enable `Object.entries` & `Object.fromEntries`. Northstar does not support IE for a long time already.